### DISCUSSION TITLE
Podcasts support on yTuner

### DISCUSSION THREAD

**Discussion by arunprasaddoss** [coffeegreg/YTuner#29] [Ideas]
Can we have support for podcasts on the YTuner?, as most podcasts has http service.
We can see the list of podcasts and consecutive episodes like we do in the Radio Browser. It will open whole new functionality to the AVR.

**Comment by coffeegreg** [comment]
:thinking:  Any examples?

**Comment by arunprasaddoss** [comment]
Not sure of many things but I am listening to the following regularly:
1. http://feed.syntax.fm/
2. http://feed.podbean.com/mapscaping/feed.xml
